### ISSUE TITLE
Requirements.txt

### ISSUE THREAD

**Issue by shweta-cmyk** [shweta-cmyk/Real-Time-Face-Mask-Detection-Using-CNN#2]
absl-py==2.3.1
astunparse==1.6.3
certifi==2025.11.12
charset-normalizer==3.4.4
flatbuffers==25.12.19
gast==0.7.0
google-pasta==0.2.0
grpcio==1.76.0
h5py==3.15.1
idna==3.11
keras==3.12.0
libclang==18.1.1
Markdown==3.10
markdown-it-py==4.0.0
MarkupSafe==3.0.3
mdurl==0.1.2
ml_dtypes==0.5.4
namex==0.1.0
numpy==2.2.6
opencv-python==4.12.0.88
opt_einsum==3.4.0
optree==0.18.0
packaging==25.0
pillow==12.0.0
protobuf==6.33.2
Pygments==2.19.2
requests==2.32.5
rich==14.2.0
scipy==1.15.3
six==1.17.0
tensorboard==2.20.0
tensorboard-data-server==0.7.2
tensorflow==2.20.0
termcolor==3.3.0
typing_extensions==4.15.0
urllib3==2.6.2
Werkzeug==3.1.4
wrapt==2.0.1
